x-pack/plugins/observability/server/ui_settings.ts

```diff
@@ -354,4 +355,32 @@ export const uiSettings: Record<string, UiSettings> = {
     requiresPageReload: true,
     type: 'boolean',
   },
+  [syntheticsThrottlingEnabled]: {
+    category: [observabilityFeatureId],
+    name: i18n.translate('xpack.observability.syntheticsThrottlingEnabledExperimentName', {
+      defaultMessage: 'Enable Synthetics throttling (Experimental)',
+    }),
+    value: false,
+    description: i18n.translate(
+      'xpack.observability.syntheticsThrottlingEnabledExperimentDescription',
+      {
+        defaultMessage:
+          'Enable Synthetics throttling UI configurations. Throttling is disabled by default. On Elastic managed infrastructure public location, throttling has some issues, {link}',
+        values: {
+          link: throttlingDocsLink({
+            href: 'https://github.com/elastic/synthetics/blob/main/THROTTLING.md',
```

Comment:
@shahzad31 Can you update this
```suggestion
            href: 'https://github.com/elastic/synthetics/blob/main/docs/throttling.md',
```